webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -0,0 +1,268 @@
+        }
+
+        return (
+            <>
+                <GitlabLabelSelector
+                    projectID={this.state.project?.project_id}
+                    projectName={this.state.project.name}
+                    theme={this.props.theme}
+                    selectedLabels={this.state.labels}
+                    onChange={this.handleLabelsChange}
+                />
+
+                <GitlabAssigneeSelector
+                    projectID={this.state.project?.project_id}
+                    projectName={this.state.project.name}
+                    theme={this.props.theme}
+                    selectedAssignees={this.state.assignees}
+                    onChange={this.handleAssigneesChange}
+                />
+
+                <GitlabMilestoneSelector
+                    projectID={this.state.project?.project_id}
+                    projectName={this.state.project.name}
+                    theme={this.props.theme}
+                    selectedMilestone={this.state.milestone}
+                    onChange={this.handleMilestoneChange}
+                />
+            </>
+        );
+    }
+
+    render() {
+        if (!this.props.visible) {
+            return null;
+        }
+
+        const theme = this.props.theme;
+        const {error, submitting} = this.state;
+        const style = getStyle(theme);
+
+        const requiredMsg = 'This field is required.';
+        let issueTitleValidationError = null;
+        if (this.state.showErrors && !this.state.issueTitleValid) {
+            issueTitleValidationError = (
+                <p className='help-text error-text'>
+                    <span>{requiredMsg}</span>
+                </p>
+            );
+        }
+
+        let submitError = null;
+        if (error) {
+            submitError = (
+                <p className='help-text error-text'>
+                    <span>{error}</span>
+                </p>
+            );
+        }
+
+        const component = (
+            <div>
+                <GitlabProjectSelector
+                    onChange={this.handleProjectChange}
+                    value={this.state.project && this.state.project.name}
+                    required={true}
+                    theme={theme}
+                    addValidate={this.validator.addComponent}
+                    removeValidate={this.validator.removeComponent}
+                />
+
+                <Input
+                    id={'title'}
+                    label='Title for the GitLab Issue'
+                    type='input'
+                    required={true}
+                    disabled={false}
+                    maxLength={MAX_TITLE_LENGTH}
+                    value={this.state.issueTitle}
+                    onChange={this.handleIssueTitleChange}
+                />
+                {issueTitleValidationError}
+
+                {this.renderIssueAttributeSelectors()}
+
+                <Input
+                    label='Description for the GitLab Issue'
```

Comment:
```suggestion
                    label='Issue description'
```
it's already inside a Gitlab plugin, no need to mentioned it